app/src/main/kotlin/com/wire/android/ui/home/conversations/messagedetails/MessageDetailsReadReceipts.kt

```diff
@@ -64,6 +65,7 @@ fun MessageDetailsReadReceipts(
     }
 }
 
+@MultipleThemePreviews
```

Comment:
I think when adding MultipleThemePreviews we should wrap the preview in WireTheme {}